lib/screens/surveydetails/widget/question_paging_widget.dart

```diff
@@ -28,80 +30,96 @@ class _QuestionPagingWidgetState extends State<QuestionPagingWidget> {
 
   @override
   Widget build(BuildContext context) {
+    if (widget.surveyDetailUiModel.questions.isEmpty) {
+      return const SizedBox.expand();
+    }
     final PageController pageController = PageController();
-    return Stack(
-      children: [
-        PageView(
-          controller: pageController,
-          onPageChanged: _onPageChanged,
-          physics: const NeverScrollableScrollPhysics(),
-          children: widget.questions
-              .map((QuestionUiModel question) => BackgroundWidget(
-                    image: Image.network(question.largeCoverImageUrl).image,
-                  ))
-              .toList(),
-        ),
-        _buildQuestionContent(
-            context, widget.questions[_currentIndex], pageController)
-      ],
-    );
-  }
 
-  Widget _buildQuestionContent(
-    BuildContext context,
-    QuestionUiModel question,
-    PageController controller,
-  ) {
     return Stack(
       children: [
+        BackgroundWidget(
+          image: Image.network(widget.surveyDetailUiModel.largeCoverImageUrl)
+              .image,
+        ),
         SafeArea(
           child: Padding(
             padding: const EdgeInsets.all(20),
             child: Column(
               crossAxisAlignment: CrossAxisAlignment.start,
               children: [
-                _buildQuestionToolbar(context, question),
-                const SizedBox(height: Dimensions.paddingLarge),
-                _buildQuestionHeader(context, question),
+                _buildQuestionToolbar(
+                  context,
+                  widget.surveyDetailUiModel.questions[_currentIndex],
+                ),
+                _pagedQuestions(context, pageController),
                 const Spacer(),
-                _buildQuestionFooter(context, question, controller),
+                _buildQuestionFooter(
+                  context,
+                  widget.surveyDetailUiModel.questions[_currentIndex],
+                  pageController,
+                ),
               ],
             ),
           ),
-        )
+        ),
       ],
     );
   }
 
+  Widget _pagedQuestions(
```

Comment:
missing the verb as the prefix of the function naming convention 😄 
```suggestion
  Widget _buildPagedQuestions(
```